src/SourceBuild/content/eng/allowed-sb-binaries.txt

```diff
@@ -23,33 +23,19 @@
 **/*.icm
 **/*.reg
 
-**/testCert*.pfx
-**/TestCert*.pfx
-
-eng/common/loc/*.lss # UTF16-LE text files
```

Comment:
Are these eng/common .lss no longer flagged as binary files is that why they are removed?

Reply:
Yes, the tooling is not detecting them as binaries. The files currently present in the VMR are:

```
./src/sdk/eng/common/loc/P22DotNetHtmlLocalization.lss
./src/wpf/eng/common/loc/P22DotNetHtmlLocalization.lss
./src/installer/eng/common/loc/P22DotNetHtmlLocalization.lss
./src/installer/eng/loc/wxl_loc.lss
./src/cecil/eng/common/loc/P22DotNetHtmlLocalization.lss
./src/xdt/eng/common/loc/P22DotNetHtmlLocalization.lss
./src/fsharp/eng/common/loc/P22DotNetHtmlLocalization.lss
./src/razor/eng/common/loc/P22DotNetHtmlLocalization.lss
./src/arcade/eng/common/loc/P22DotNetHtmlLocalization.lss
./src/windowsdesktop/eng/common/loc/P22DotNetHtmlLocalization.lss
./src/test-templates/eng/common/loc/P22DotNetHtmlLocalization.lss
./src/deployment-tools/eng/common/loc/P22DotNetHtmlLocalization.lss
./src/scenario-tests/eng/common/loc/P22DotNetHtmlLocalization.lss
./src/templating/eng/common/loc/P22DotNetHtmlLocalization.lss
./src/sourcelink/eng/common/loc/P22DotNetHtmlLocalization.lss
./src/source-build-reference-packages/eng/common/loc/P22DotNetHtmlLocalization.lss
./src/command-line-api/eng/common/loc/P22DotNetHtmlLocalization.lss
./src/msbuild/eng/common/loc/P22DotNetHtmlLocalization.lss
./src/roslyn-analyzers/eng/common/loc/P22DotNetHtmlLocalization.lss
./src/winforms/eng/common/loc/P22DotNetHtmlLocalization.lss
./src/symreader/eng/common/loc/P22DotNetHtmlLocalization.lss
./src/runtime/eng/common/loc/P22DotNetHtmlLocalization.lss
./src/vstest/eng/common/loc/P22DotNetHtmlLocalization.lss
./src/roslyn/eng/common/loc/P22DotNetHtmlLocalization.lss
./src/aspnetcore/eng/common/loc/P22DotNetHtmlLocalization.lss
./src/aspire/eng/common/loc/P22DotNetHtmlLocalization.lss
./src/emsdk/eng/common/loc/P22DotNetHtmlLocalization.lss
./src/source-build-externals/eng/common/loc/P22DotNetHtmlLocalization.lss
./src/diagnostics/eng/common/loc/P22DotNetHtmlLocalization.lss
./eng/common/loc/P22DotNetHtmlLocalization.lss
```

Investigating them, they are classified as "XML 1.0 document, ASCII text, with very long lines (573)", so not binaries.